src/styles/themes/_dark-theme.scss

```diff
@@ -0,0 +1,751 @@
+body[dark] {
```

Comment:
move themable color vars from `_default-theme.scss` to a new `_light-theme.scss` file? Perhaps keep the variables with persistent values in `_default-theme.scss` as an optimization?

Reply:
Addressed will push update.